content/install_chef_air_gap.md

```diff
@@ -161,9 +161,11 @@ custom template, and then modify the package and Ruby gem sources.
     template file name, be sure to replace `airgap.erb` with the
     template file you created during the last step.
 
-4. Update `airgap.erb` to replace `omnitruck.chef.io` with the URL of
+4. Update `airgap.erb` to replace `https://chefdownload-commericial.chef.io/install.sh?license_id=<YOUR LICENSE ID>` with  the URL of
```

Comment:
Is this hostname correct, or does it contain a typo? It appears to misspell `commercial` as `commericial` (note extra I) This mistake is throughout this PR and https://github.com/inspec/inspec/pull/7118 . Please advise and correct if needed.

Reply:
Yes, there was a typo error, and we have updated the URL to `chefdownload-commerical.chef.io`. 
Note: The hosted URL also has a typo (`commerical` instead of `commercial`) which cannot be modified